inc/update.php

```diff
@@ -514,217 +500,6 @@ function largo_update_custom_less_variables() {
 	}
 }
 
-/**
- * Checks for use of deprecated widgets and posts an alert
- */
-function largo_check_deprecated_widgets() {
```

Comment:
@rclations could you resubmit this PR retaining (for now) the deprecation stuff and just removing the widget files/css/etc.? The update from, say, 0.3 to 0.5.5 is a problem we actually have (for a few sites) so I'm ok with being a little conservative here and ditching this later.